### ISSUE TITLE
PG 14, PG 15 support

### ISSUE THREAD

**Issue by a-mckinley** [launchql/libpg-query-node#21]
I ran into this issue using a vscode extension Plpgsql language server, https://github.com/UniqueVision/plpgsql-lsp/.

PG14 introduced "create or replace trigger" which causes a parse error in libpg-query-node.

Do you intend to introduce support at some point for PG14?

**Comment by pyramation** [comment]
Hi! (pardon the delay, I just fixed my notifs)

https://github.com/pganalyze/libpg_query still is on 13, and therefore so are we. We can track and upgrade when they do.

**Comment by tgensol** [comment]
Hi there =) 

https://github.com/pganalyze/libpg_query is now for 14 (and 15), thanking you in advance

**Comment by pyramation** [comment]
tracking [14 here](https://github.com/pyramation/libpg-query-node/pull/26) and [15 here](https://github.com/pyramation/libpg-query-node/pull/27)

**Comment by pyramation** [comment]
- [13.3.1](https://www.npmjs.com/package/libpg-query/v/13.3.1)
- [14.0.0](https://www.npmjs.com/package/libpg-query/v/14.0.0)
- 15.0.0 (in progress)

**Comment by nith2001** [comment]
Hello @pyramation, thank you for the work you've been doing in `libpg-query-node`. I'm a developer from the Supabase codebase and we've been facing issues parsing Postgres queries past V13. We were able to trace the issue here since we use [pgsql-parser](https://github.com/pyramation/pgsql-parser), which uses this library as a sub-dependency.

We noticed that you have the V15 PR ready [here](https://github.com/pyramation/libpg-query-node/pull/27), and we wanted to ask if it would be possible to merge this in, assuming it's ready. If not, could we get a timeline on how long before it can be merged?
